### DISCUSSION TITLE
Is the diagram view always static while filtering?

### DISCUSSION THREAD

**Discussion by soerendomroes** [eclipse-glsp/glsp#1211] [Q&A] [answered]
In the MDEnet tutorial it was mentioned that the filtered view was not editable.
https://youtu.be/f5gVKh04Oho?t=4863

Is this generally not possible to edit filtered views using the GLSP, or this is just not implemented for the demo feature?

**Comment by planger** [comment, marked as answer]
No this is just implemented this way in the demo.
Of course there are some conceptual questions with respect to the layout. If you apply auto-layouting for layouting the filtered view, how do you handle moving an element with respect to the original layout before the filtering and auto-layouting. But aside from that there is no technical restrictions to not make this filtered view editable to some degree.
